katello.gemspec

```diff
@@ -43,7 +43,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "runcible", ">= 2.11.0", "< 3.0.0"
   gem.add_dependency "anemone"
   gem.add_dependency "pulpcore_client"
-  gem.add_dependency "pulp_file_client", "3.0.0rc2.dev.1558441126"
+  gem.add_dependency "pulp_file_client", "< 1.0"
```

Comment:
FYI: https://github.com/Katello/katello/commit/f4a7b6bb1347e05a648374a06ae41c29338d9518 already did this.